multiqc/multiqc.py

```diff
@@ -981,7 +981,7 @@ def __rich_measure__(self, console: rich.console.Console, options: rich.console.
                     logger.error("Output directory {} already exists.".format(config.plots_dir))
                     logger.info("Use -f or --force to overwrite existing reports")
                     shutil.rmtree(tmp_dir)
-                    sys.exit(1)
+                    return {"report": report, "config": config, "sys_exit_code": 1}
```

Comment:
I guess because of these lines https://github.com/ewels/MultiQC/blob/master/multiqc/multiqc.py#L933-L934 it also should never happen, unless a folder was created by another process. Simulated that, works fine

Reply:
Yes, you might be right there. I suspect that this code predates the rotating suffix thing being added. In which case, we can presumably just delete this chunk of code.